src/fuzzers/generic-analyzer-fuzzer.cc

```diff
@@ -40,6 +40,12 @@ static zeek::analyzer::Analyzer* add_analyzer(zeek::Connection* conn)
 	auto* tcp = new zeek::packet_analysis::TCP::TCPSessionAdapter(conn);
 	auto* pia = new zeek::analyzer::pia::PIA_TCP(conn);
 	auto a = zeek::analyzer_mgr->InstantiateAnalyzer(TOSTRING(ZEEK_FUZZ_ANALYZER), conn);
+	if ( ! a )
+		{
+		printf("Unknown or unsupported analyzer %s found\n", TOSTRING(ZEEK_FUZZ_ANALYZER));
```

Comment:
Stderr for errors?

`std::fprintf(stderr, ...)` or `std::cerr` ?

Instead of `abort()` which raises signals and triggers coredumps depending on the configuration, `exit(1)` might just be enough here, too?